### PR TITLE
move connectionpool to jdbc Connection from driver

### PR DESCRIPTION
**Pull request by Nicole00**

### CODE REVIEW

**Comment by Nicole00** [comment]
The Driver and Connection is circularly dependent, and driver should not maintain the Nebula connection pool.

If driver maintain the connection pool, when users use `getConnection(url, user, password)` to connect another nebula graph address, there would be connection mistakes, because the connection pool has been bind to the original address.

**Review by wey-gu — DISMISSED** [review]

File: src/main/java/com/vesoft/nebula/jdbc/impl/NebulaDriver.java [hunk 1]
```diff
@@ -39,119 +37,80 @@ public class NebulaDriver extends NebulaAbstractDriver {
...
 
     /**
      * connect to the NebulaGraph
      */
     @Override
-    public Connection connect(String url, Properties connectionConfig) throws SQLException {
+    public Connection connect(String url, Properties properties) throws SQLException {
         if (!acceptsURL(url)) {
             throw new SQLException("url: " + url + " is not accepted, " +
                     "url example: jdbc:nebula://graphSpace " +
```
Comment (xiajingchun) [inline src/main/java/com/vesoft/nebula/jdbc/impl/NebulaDriver.java]: url example can be changed to jdbc:nebula://host_ip:port,host_ip2:port...//graphSpace

**Review by xiajingchun — CHANGES_REQUESTED** [review]

File: src/main/java/com/vesoft/nebula/jdbc/impl/NebulaDriver.java (hunk 1 above)
Comment (Nicole00) [reply src/main/java/com/vesoft/nebula/jdbc/impl/NebulaDriver.java]: Right, good catch

**Comment by Nicole00** [comment]
this pr has been tested with Example, more unit tests will be added in the next or the next.next pr. @wey-gu @xiajingchun

**Review by wey-gu — APPROVED** [review]
👍

**Review by xiajingchun — APPROVED** [review]

**Pull request merged**